packages/db/migrations/0028_watery_clea.sql

```
@@ -0,0 +1,17 @@
+	"owner_user_id" text NOT NULL,
+	"thread_id" text NOT NULL,
+	"sandbox_id" text NOT NULL,
+	"status" text DEFAULT 'active' NOT NULL,
+	"staged_datasets" jsonb NOT NULL,
+	"created_at" bigint NOT NULL,
+	"last_used_at" bigint NOT NULL,
+	CONSTRAINT "analysis_sandboxes_status_check" CHECK ("analysis_sandboxes"."status" in ('active', 'deleted'))
+);
+--> statement-breakpoint
+ALTER TABLE "artifacts" DROP CONSTRAINT "artifacts_artifact_type_check";--> statement-breakpoint
+ALTER TABLE "analysis_sandboxes" ADD CONSTRAINT "analysis_sandboxes_owner_user_id_users_owner_user_id_fk" FOREIGN KEY ("owner_user_id") REFERENCES "public"."users"("owner_user_id") ON DELETE cascade ON UPDATE no action;--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)